src/langchain_google_alloydb_pg/vectorstore.py

```diff
@@ -57,6 +57,7 @@ def __init__(
         fetch_k: int = 20,
         lambda_mult: float = 0.5,
         index_query_options: Optional[QueryOptions] = None,
+        relevance_score_fn: Optional[Callable[[float], float]] = None,
```

Comment:
It's not worth supporting an override in the relevance score

Reply:
deleted